.travis.yml

```diff
@@ -31,6 +31,17 @@ deploy:
       LOG_LEVEL: debug
       SQLITE_FILE: checkInCards.sql
       SQLITE_BUCKET: check-in-card-data-qa
```

Comment:
We should be able to delete these lines from all the environments in the travis file. (also where they show up in the SAM stuff and the test suite).